src/theory/datatypes/theory_datatypes.cpp

```diff
@@ -382,8 +382,9 @@ void TheoryDatatypes::postCheck(Effort level)
   }
 
   Trace("datatypes-check") << "Finished check effort " << level << std::endl;
-  if( Debug.isOn("datatypes") || Debug.isOn("datatypes-split") ) {
-    Notice() << "TheoryDatatypes::check(): done" << endl;
+  if (isVerboseOn(1))
```

Comment:
Let's just change this to Debug("datatypes"), im not sure why Notice() was used here.